docs/agent-guard-mcp-proxy.md

```diff
@@ -0,0 +1,90 @@
+
+<p style="text-align: center;">
+    <img src="https://raw.githubusercontent.com/cyberark/agent-guard/refs/heads/main/resources/logo.png" alt="agentwatch - AI Agent Observability Platform" width="400"/>
+    
+</p>
+<h3 style="font-family: 'Fira Mono', Monospace; text-align: center;">MCP Proxy for AI agents</h3>
+
+## 🌟 Overview
+
+CyberArk's Agent Guard MCP Proxy is an AI agent security tool built for developers and has full auditing and monitoring capabilities. Every interaction between the AI agent and your MCP servers is logged, providing complete traceability and compliance with enterprise security standards.
+
+The tool uses the [Agent Guard CLI](../agent_guard_core/cli.md).
+
+The Docker image, `<name>`, is available from the [AWS Marketplace](https://link.to.aws.marketplace.com).  **!!!NEED TO UPDATE IMAGE NAME ADD LINK TO AWS MARKETPLACE!!!!.**
```

Comment:
This is a live repo, so I wouldn't want it to merge as is. Is this line a blocker for merging until we update it?